rotkehlchen/globaldb/schema.py

```diff
@@ -238,6 +238,16 @@
 );
 """
 
+DB_CREATE_GENERAL_CACHE = """
+CREATE TABLE IF NOT EXISTS general_cache (
+    type CHAR(1) NOT NULL,
+    name TEXT NOT NULL,
+    value TEXT,
+    last_queried_ts INTEGER,
+    UNIQUE(type, name)
```

Comment:
Why just unique? This should be the primary key! Otherwise you are creating an identifier as a primary key which is just invisible in the schema.

EDIT: After reading the code more and understanding how you use it, the primary key should be type, name, value.

So you can have multiple values per key. The downside of this is that when you want to update a key you should first delete all its values.